Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -996,13 +996,14 @@ def rasterize_email_command():  # pragma: no cover
 
     navigation_timeout = int(demisto.args().get('max_page_load_time', DEFAULT_PAGE_LOAD_TIME))
 
-    with open('htmlBody.html', 'w', encoding='utf-8-sig') as f:
-        f.write(f'<html style="background:white";>{html_body}</html>')
-
-    path = f'file://{os.path.realpath(f.name)}'
-
-    rasterize_output = perform_rasterize(path=path, rasterize_type=rasterize_type, width=width, height=height,
-                                         offline_mode=offline, navigation_timeout=navigation_timeout, full_screen=full_screen)
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.html', encoding='utf-8-sig') as tf:
+        demisto.debug(f'{html_body=}')
+        tf.write(f'<html style="background:white";>{html_body}</html>')
+        tf.flush()
+        path = f'file://{os.path.realpath(tf.name)}'
+        demisto.debug(f'{path=}')
+        rasterize_output = perform_rasterize(path=path, rasterize_type=rasterize_type, width=width, height=height,
+                                             offline_mode=offline, navigation_timeout=navigation_timeout, full_screen=full_screen)
 
     res = fileResult(filename=file_name, data=rasterize_output[0][0])
```

Comment:
`rasterize_output` is `None` if there were problems with the temp file.
```suggestion
        res = fileResult(filename=file_name, data=rasterize_output[0][0])
```